lib/iris/tests/integration/experimental/test_CubeRepresentation.py

```diff
@@ -136,12 +137,12 @@ def test_identfication(self):
 
     def test_header__name(self):
         header = self.representer._make_header()
-        expected_name = self.cube.name().title().replace('_', ' ')
+        expected_name = escape(self.cube.name().title().replace('_', ' '))
```

Comment:
Why did you use `escape` here? self.cube is one we create and shouldn't have a name that needs sanitising

Reply:
The thinking here is that I want the process to be consistent with what is happening in the code it is testing, even if `escape` isn't doing anything in this instance.